kademlia/protocol.py

```diff
@@ -108,9 +116,9 @@ def handleCallResponse(self, result, node):
         """
         if result[0]:
             self.log.info("got response from %s, adding to router" % node)
-            self.router.addContact(node)
             if self.router.isNewNode(node):
```

Comment:
Yeah we can do that. The addContact call always follows that method as well, do you think that should be put in that consolidated method as well or keep it separate?